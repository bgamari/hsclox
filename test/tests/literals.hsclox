print 1;
print true;
print false;
print nil;
print "hi";
