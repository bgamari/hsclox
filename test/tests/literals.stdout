== code ==
0000    1 OP_CONSTANT         0 '1
0002    | OP_PRINT
0003    2 OP_TRUE
0004    | OP_PRINT
0005    3 OP_FALSE
0006    | OP_PRINT
0007    4 OP_NIL
0008    | OP_PRINT
0009    5 OP_CONSTANT         1 'hi
0011    | OP_PRINT
0012    6 OP_RETURN
1
true
false
nil
hi
